src/features/about-video/about-video.tsx

```diff
@@ -2,12 +2,22 @@ import { Title } from '@/app/components';
 
 import './about-video.scss';
 
-type AboutVideoProps = { language?: 'en' | 'ru' };
-export const AboutVideo = ({ language = 'en' }: AboutVideoProps) => {
+type AboutVideoProps = { lang?: 'en' | 'ru' };
+
+const LocalizedContent = {
```

Comment:
```suggestion
const localizedContent = {
```
Please fix in all code